### ISSUE TITLE
DateComponents addition?

### ISSUE THREAD

**Issue by toshi0383** [malcommac/SwiftDate#445]
I confirmed `&&` operator works against DateComponents. It's awesome!
https://github.com/malcommac/SwiftDate/issues/348

But I feel a little confused when I use `+` operator between DateComponents.
```
let date = Date()
let timeInterval = date - (date - (1.hour + 1.minute)) // got 0 😕
let timeInterval = date - (date - (61.minute)) // got 3660
let timeInterval = date - (date - (1.hour && 1.minute)) // got 3660
```

Is this expected behavior? If so, why?

**Comment by malcommac** [comment]
Thank you for your report. It's not the correct behaviour but I've fixed it inside the new 4.1.6 build and added your as a unit test.
This is the reference commit https://github.com/malcommac/SwiftDate/commit/057aafb75b1367b7fb386ff2557fa857418b3a23.
Again thank you.

**Comment by toshi0383** [comment]
Thank you for quick fix!😄
